### PR TITLE
Create simple nio.conf and standalone.conf files

### PR DESCRIPTION
**Pull request by mattdodge**
This moves the old, full nio.conf file to `nio.conf.example` for reference of available options. Now, nio.conf provides a simple conf file that can be used to run your nio instance as a pubkeeper client. If you want to run your nio instance with a stanalone pubkeeper server, just include the `standalone.conf` file as an additional configuration file.

Basic client mode
```
niod
```

Stanalone pk server mode
```
niod -s nio.conf -s standalone.conf
```

### CODE REVIEW

**Comment by mattdodge** [comment]
@hansmosh @tyoungNIO what do you guys think of this approach?

**Review by tlugger — APPROVED** [review]

File: nio.conf.example [hunk 1]
```diff
@@ -0,0 +1,673 @@
...
+# ports used by master broker to communicate with its clients
+XPUB_PORT=9000
+XSUB_PORT=9001
+
+
+[environment]
+
+# Location of custom extensions for auto-discovery
+#
+#blocks=blocks
```
Comment (shadetree01010100) [inline nio.conf.example:78]: Should all of these default values be un-commented in the example?

**Review by shadetree01010100 — APPROVED** [review]
I like this a lot, the only thing that jumped out at me is the default values are still commented out in the example. Probably doesn't matter, but it isn't necessary anymore.

File: nio.conf.example (hunk 1 above)
Comment (mattdodge) [reply nio.conf.example:78]: Hmm that's an interesting thought. I'll check around some other places and see what best practices look like

Comment (hansmosh) [reply nio.conf.example:78]: What I've seen before is default values are shown in the file but commented out. Like how we have it.

**Pull request merged**